app/mailers/core_application_mailer.rb

```diff
@@ -0,0 +1,55 @@
+require 'net/http'
+require 'uri'
+require 'json'
+
+class CoreApplicationMailer < ActionMailer::Base
+  layout "mailer"
+
+  def send_custom_email(recipient:, subject:, body_html:)
+    # Get the token form the cloud_admin instance
+    ###################################
+    # TODO: add to the cloud_admin user "role:cloud_resource_admin or role:resource_service"
```

Comment:
What does it mean? Do we still have to do something?

Reply:
We need to use the seeder to add the Dasboard user on of this roles:
```
role:cloud_resource_admin or role:resource_service
```
for ccadmin/cloud_admin